tests/primeqa/services/rest_server/test_server.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 PrimeQA Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+from unittest.mock import MagicMock
+import pytest
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+# from primeqa.services.rest_server.server import FastAPI
+import sys
```

Comment:
Quick note: It is recommended practice to first import python's built-in libraries like `sys` or `os` before importing 3rd party libraries like `fastapi`. The last imports are usually about packages in your code back.